common/daemon.h

```diff
@@ -63,7 +64,7 @@ namespace creds_fetcher
         {
             if ( level >= log_level )
             {
-                sd_journal_print( level, fmt, logs... );
+                //sd_journal_print( level, fmt, logs... );
```

Comment:
This has to be uncommented, or add a TODO for now.

Reply:
Addressed in latest updated PR